cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -278,6 +281,7 @@ public void testMaximumActiveTasks() {
       OperationFuture future2 =
           userTaskManager.getOrCreateUserTask(mockHttpServletRequest2, mockHttpServletResponse, uuid -> future, 0, true, null).get(0);
       Assert.assertEquals(future, future2);
+      EasyMock.verify(mockHttpServletRequest2, mockHttpServletResponse);
```

Comment:
Should this be `EasyMock.verify(mockHttpSession2, mockHttpServletRequest2, mockHttpServletResponse);`?